metadata.json

```diff
@@ -64,6 +64,12 @@
       "operatingsystem": "SLED"
     },
     {
+      "operatingsystem": "OpenSuSE",
```

Comment:
Question to all three modules I sent pull requests, could it stay for the time being, until someone in the future runs into potential conflicts with it? Or you want to have it changed now, so it's easier in the future. If you want to change it now, I'd combine SLES/SLED, and add OpenSuSE as the "other". Getting the specs adapted, might complicate matter a bit ;)